### ISSUE TITLE
Contributor links in contributor page should lead to contributor items only

### ISSUE THREAD

**Issue by jacobthill** [sul-dlss/dlme#1349]
If you click on a contributor in the [contributors page](https://dlmenetwork.org/library/statistics) it leads to a search result for all objects instead of only the objects from that contributor.
<img width="906" alt="Screen Shot 2021-11-08 at 11 51 21 AM" src="https://user-images.githubusercontent.com/37662787/140784132-c9fe7ff0-d639-48a9-97a4-ed44161ecea5.png">

**Comment by jacobthill** [comment]
Is this supposed to be working in dev or are we waiting on something? I assume its supposed to be working since its in the QA column but let me know if not. Its not currently working. Clicking on 'American Numismatic Society' under 'Data Contributors' produces this url: https://dev.dlmenetwork.org/library/catalog?f%5Bagg_provider.en_ssim%5D%5B%5D=American+Numismatic+Society which returns all resources in the database, instead of only those with agg_provider="American Numismatic Society".

**Comment by corylown** [comment]
This is change is merged into the main branch. I do no believe it is deployed anywhere. I think we need to have some team agreement about process to avoid confusion like this.

**Comment by jacobthill** [comment]
Found a new bug related to items by type counts, opened https://github.com/sul-dlss/dlme/issues/1436

**Comment by corylown** [comment]
The fix for this issue is merged into `main`, deployed to dev, and is working so I am closing this again.

**Comment by mwerla** [comment]
@jacobthill Looks good to me
